### ISSUE TITLE
Rawhide SELinux test failures

### ISSUE THREAD

**Issue by jelly** [cockpit-project/cockpit-machines#1521]
Likely SELinux related https://artifacts.dev.testing-farm.io/959c2d9f-003f-4ae4-9bdb-d044dfee9bac/

**Comment by martinpitt** [comment]
Still happen today: https://artifacts.dev.testing-farm.io/41403e3d-bb20-4f13-99bf-4b547753d8fd/

**Comment by mvollmer** [comment]
I think the intention is to skip SELinux audit messages in TF runs. plans/all.fmf has this:

```
# Let's handle them upstream only, don't break Fedora/RHEL reverse dependency gating
environment:
    TEST_AUDIT_NO_SELINUX: 1
```

but the actual test run still has it empty:

```
TEST_ALLOW_JOURNAL_MESSAGES: 
TEST_AUDIT_NO_SELINUX: 
```

**Comment by mvollmer** [comment]
Let's try to pass the environment down into the container started by browser.sh: https://github.com/cockpit-project/cockpit-machines/pull/1530

**Comment by martinpitt** [comment]
Fixed in #1530
